package.json

```diff
@@ -33,7 +33,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "html-webpack-plugin": "^5.5.3",
     "mini-css-extract-plugin": "^2.7.6",
-    "openapi-typescript": "^6.4.0",
+    "openapi-typescript": "5.4.1",
```

Comment:
FYI @jossi87 - until/unless we bump the Swagger dep on the server side, we can't bump this to v6